### ISSUE TITLE
What's the performance for teacher model trained on the VKITTIv2 and hypersim

### ISSUE THREAD

**Issue by Kai-Yin-UoA** [DepthAnything/Depth-Anything-V2#288]
I am currently running the training process for teacher model on vkittiv2, but the model's performance is not the same level of final models, as shown in the log below. 

Despite other different configurations on training, I wonder what is the training result for teacher's model on synthetic models. I have read throught the paper and only find the relation between the impact of synthetic data on the final adaptation performance. 

**Is there any performance simply on synthetic models can be shared.** 

I wonder what level of accuracy support further fine-tuning on student models. ( I am definitly sure it must be higher than the peformance displayed in the paper. )

```bash
[2025-07-28 09:52:15,413][    INFO] ==========================================================================================
[2025-07-28 09:52:15,413][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2025-07-28 09:52:15,414][    INFO]    0.834,    0.967,    0.994,    0.129,    0.653,    4.004,    0.172,    0.052,    0.161
[2025-07-28 09:52:15,414][    INFO] ==========================================================================================
```
